libs/sdk-model/src/insight/index.ts

```diff
@@ -663,6 +672,49 @@ export function insightReduceItems<T extends IInsightDefinition>(
     } as T;
 }
 
+/**
+ * @public
```

Comment:
Please add some docs here (copy paste from the function that gets this).

Reply:
fixed